testnet/all-config.rst

```diff
@@ -0,0 +1,21 @@
+.. _testnet-all-config:
+
+=========================
+All configuration options
```

Comment:
I don't think it's very enlightening as a title in the panel. Maybe Node and Whitelist configuration ?

Reply:
This could be a place for all configurations. For example [quarkus](https://quarkus.io/guides/all-config)

Reply:
done